### PR TITLE
Support macOS v12

### PR DESCRIPTION
**Pull request by andrewtheis**
I tested this on one of my apps using a macOS 12 VM (easy to do w/[VirtualBuddy](https://github.com/insidegui/VirtualBuddy)). Appears everything works, but obviously just one use case.

### CODE REVIEW

File: Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift [hunk 1]
```diff
@@ -36,7 +36,9 @@ final class FluidMenuBarExtraWindow<Content: View>: NSPanel {
     private lazy var hostingView: NSHostingView<some View> = {
         let view = NSHostingView(rootView: rootView)
         // Disable NSHostingView's default automatic sizing behavior.
-        view.sizingOptions = []
+        if #available(macOS 13.0, *) {
+            view.sizingOptions = []
+        }
```
Comment (lfroms) [inline Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift:41]: What are the implications of these changes? Any changes to behavior?

File: Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift [hunk 2]
```diff
@@ -64,7 +66,11 @@ final class FluidMenuBarExtraWindow<Content: View>: NSPanel {
         titlebarAppearsTransparent = true
 
         animationBehavior = .none
-        collectionBehavior = [.auxiliary, .stationary, .moveToActiveSpace, .fullScreenAuxiliary]
+        if #available(macOS 13.0, *) {
+            collectionBehavior = [.auxiliary, .stationary, .moveToActiveSpace, .fullScreenAuxiliary]
+        } else {
+            collectionBehavior = [.stationary, .moveToActiveSpace, .fullScreenAuxiliary]
+        }
```
Comment (lfroms) [inline Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift:73]: Same here, how does this affect behavior (i.e. animations) on macOS 12? Do you have a screen recording of the result?

File: Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift (hunk 1 above)
Comment (andrewtheis) [reply Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift:41]: As far as I can tell, no. I tested on the macOS 12 VM

File: Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift (hunk 2 above)
Comment (andrewtheis) [reply Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift:73]: Yup, here ya go

https://github.com/lfroms/fluid-menu-bar-extra/assets/469799/ce0535cd-2a80-4751-b381-265fde2437e0

**Review by lfroms — APPROVED** [review]
Thanks, @andrewtheis!  These changes look good to me.

File: Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift (hunk 2 above)
Comment (lfroms) [reply Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift:73]: Cool, that seems to work well.

**Pull request merged**